lms/envs/common.py

```diff
@@ -345,9 +345,12 @@
     # Enable LTI Provider feature.
     'ENABLE_LTI_PROVIDER': False,
 
-    # Show Language selector.
+    # Show language selector in header.
     'SHOW_LANGUAGE_SELECTOR': False,
 
+    # Show language selector in footer.
+    'SHOW_FOOTER_LANGUAGE_SELECTOR': False,
```

Comment:
@andy-armstrong does this actually make sense as a setting? Would it make more sense to toggle it via waffle switch instead?